public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -122,8 +122,9 @@ function ReceiverHealth({ errorsByReceiver, someWithNoAttempt }: ReceiverHealthP
   );
 }
 const useContactPointsState = (alertManagerName: string) => {
-  const contactPointsStateRequest = useUnifiedAlertingSelector((state) => state.contactPointsState);
-  const { result: contactPointsState } = (alertManagerName && contactPointsStateRequest) || initialAsyncRequestState;
+  const { currentData: contactPointsState } = receiversApi.endpoints.contactPointsState.useQueryState({
```

Comment:
Could we use `useContactPointsStateQuery` hook?

Reply:
💯